js.js

```
@@ -0,0 +1,52 @@
+var num1 = 10,num2 = 20;
+var sum = num1+num2;
+
+if(num1<num2){console.log('Num1 is less than Num2')}else{console.log('Num1 is greater than or equal to Num2')} // bad formatting
+
+// Hardcoded API URL and credentials
+var apiUrl = "http://example.com/api";
+var apiKey = "12345";
+
+// Unused variable
+var unusedVar = 100;
```

Biome

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole module. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)